### PR TITLE
Exercise 7

### PR DESCRIPTION
**Pull request by danmousley367**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Comment by danmousley367** [comment]
I noticed the pipeline still runs on the pull request when I make changes to the `README.MD` (an ignored file), but not on push. I found [this bug report](https://github.com/actions/runner/issues/2324) with the same issue - curious to hear if this is something you've seen before, or if I'm doing something wrong?

File: .github/workflows/ci-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  push:
+    paths-ignore:
+      - '**.md'
+      - 'documentation/**'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'documentation/**'
+  schedule:
+    - cron: '0 5 * * mon'
```
Comment (JackMead) [inline .github/workflows/ci-pipeline.yml:12]: Nice addition, I do like having occasional runs scheduled  for when the app hasn't been used in a while!

File: .github/workflows/ci-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - run: echo Hello World
```
Comment (JackMead) [inline .github/workflows/ci-pipeline.yml:20]: Can we delete this entry now? 😄

**Review by JackMead — APPROVED** [review]
Nice work - simple and effective and I can see the tests happily passing in your pipeline! Go ahead and merge when you're ready 🙂

**Pull request merged**